lib/partitions_validator_utils.pm

```diff
@@ -14,14 +14,63 @@ use scheduler 'get_test_suite_data';
 use testapi;
 use Test::Assert ':all';
 use Exporter 'import';
-our @EXPORT = 'validate_partition_table';
+our @EXPORT = qw(
+  validate_partition_table
+  validate_partition_creation
+  validate_filesystem
+  validate_read_write
+  validate_unpartitioned_space);
 
 sub validate_partition_table {
     my $args = shift;
     return if check_var('BACKEND', 's390x');    # blkid output does not show partition table for dasd
     record_info("Check $args->{table_type}", "Verify if partition table type is $args->{table_type}");
     my $table_type = (split(/\"/, script_output("blkid $args->{device}")))[-1];    # last element of output eg "gpt"
-    assert_equals($args->{table_type}, $table_type, "Partition table type does not correspond to the expected one.");
+    my $converter  = {'msdos' => 'dos'};
+    my $expected   = $args->{table_type};
+    if (exists $converter->{$args->{table_type}}) { $expected = $converter->{$args->{table_type}} }
+    assert_equals($expected, $table_type, "Partition table type does not correspond to the expected one.");
+}
+
+sub validate_partition_creation {
+    my $args         = shift;
+    my @lsblk_output = split(/\n/, script_output("lsblk -n"));
+    record_info("Check $args->{mount_point}", "Verify that partition $args->{mount_point} was created.");
+    my $check;
+  READ: foreach (@lsblk_output) {
+        if ($_ =~ /\Q$args->{mount_point}\E\z/) {
+            $check = 1;
+            last READ;
```

Comment:
we don't need to use `last` with label, just `last` it is obvious from the context. WDYT?

Reply:
Yes, I was running some experimental tests and for some occasions I had issues when I didn't use label, but in this case looks like it works. I will remove it.